src/Auth/Actions/articleActions.js

```diff
@@ -41,6 +60,64 @@ export const PostArticlesFailure = (error) => {
     };
 };
 
+export const likeArticlesRequest = () => {
+    return {
+        type: types.LIKE_ARTICLES_REQUEST,
+    };
+};
+
+export const likeArticlesSuccess = (index, likedArticle) => {
```

Comment:
what is index doing here, why don't you use the article id

Reply:
ok sir